release/release.go

```diff
@@ -1108,17 +1113,42 @@ func uploadRelease(v version.Version) {
 	}
 }
 
-var linuxRepoVersionsStable = map[string]string{
-	"development": "4.0.0-15-gabcde123", // any non-rc pre-release version will send the package to the "development" repo
-	"testing":     "4.0.0-rc0",          // any rc version will send the package to the "testing" repo
-	"4.3":         "4.3.0",              // any 4.3 stable release version will send the package to the "4.3" repo
-	"4.4":         "4.4.0",              // any 4.4 stable release version will send the package to the "4.4" repo
-	"4.9":         "4.9.0",              // any 4.9 stable release version will send the package to the "4.9" repo
-	"5.0":         "5.0.0",              // any 5.0 stable release version will send the package to the "5.0" repo
+type VersionWithSigningInfo struct {
+	mongoVersionNumber string
+	notaryKeyName string
+	notaryToken string
 }
 
-var linuxRepoVersionsUnstable = map[string]string{
-	"development": "4.0.0-15-gabcde123", // any non-rc pre-release version will send the package to the "development" repo
+var linuxRepoVersionsStable = map[string]VersionWithSigningInfo{
+	"development": {"4.0.0-15-gabcde123", "", ""}, // any non-rc pre-release version will send the package to the "development" repo
+	"testing":     {"4.0.0-rc0", "", ""},          // any rc version will send the package to the "testing" repo
+	"4.3":         {"4.3.0", "server-4.4", os.Getenv("SIGNING_AUTH_TOKEN_SERVER_4_4")},              // any 4.3 stable release version will send the package to the "4.3" repo
+	"4.4":         {"4.4.0", "server-4.4", os.Getenv("SIGNING_AUTH_TOKEN_SERVER_4_4")},              // any 4.4 stable release version will send the package to the "4.4" repo
+	"4.9":         {"4.9.0", "server-5.0", os.Getenv("SIGNING_AUTH_TOKEN_SERVER_5_0")},              // any 4.9 stable release version will send the package to the "4.9" repo
```

Comment:
We need to set this env variable in common.yml still, right?

Reply:
We don't since I just override them during `exec.Command`

Reply:
We don't since I just override them during exec.Command

Reply:
So it looks like the NOTARY_TOKEN and name env var only matter when we call `curator` script. Since I changed the cmd.Env before running it, the curator process will get an updated env value for these two variables, I don't think we need to change the common.yml anymore.

Reply:
But the `SIGNING_AUTH_TOKEN_SERVER_5_0` and `SIGNING_AUTH_TOKEN_SERVER_4_4` variables are from the environment of release.go. So we still need to set these so `release.go` can get these vars, right?

Reply:
Curently we do `export NOTARY_TOKEN='${signing_auth_token_server_4_4}'` in common.yml, but we need to export vars for 5.0 and 4.0.

Reply:
These are already set on the host machine I think. So if you look at `common.yml`, 
```
          echo "${signing_auth_token_server_4_4}" > ./signing_auth_token
```
it's just retrieving this value and write to file. There is no place in our code to set these values.

Reply:
I kinda get what you said, do you mean we need to have something like
```
          export NOTARY_TOKEN_4_4='${signing_auth_token_server_4_4}'
          export NOTARY_TOKEN_5_0='${signing_auth_token_server_5_0}'
```
inside common.yml and getEnv from these exported variables? Is that necessary?

Reply:
Yes, I think so. `os.Getenv()` needs to get the env variable from somewhere. So we need to set the environment variables there.

Reply:
The thing is currently we call Getenv on "signing_auth_token_server_4_4" and ""signing_auth_token_server_5_0" which should already be set on the server.  `export NOTARY_TOKEN` in `common.yml` is only for curator and is already taken care of by this piece of code 
```
						envOverrides["NOTARY_KEY_NAME"] = versionWithSigningInfo.notaryKeyName
						envOverrides["NOTARY_TOKEN"] = versionWithSigningInfo.notaryToken
....
	cmd.Env = os.Environ()
	for k, v := range envOverrides {
		cmd.Env = append(cmd.Env, fmt.Sprintf("%s=%s", k, v))
	

```

Reply:
How are they set on the server? Maybe I'm wrong here, but we have them set as Evergreen variables. But those variables are not set as environment variables on the server until we set them in the `pre` script in `common.yml`.

Reply:
It's changed to NOTARY_TOKEN_4_4 and NOTARY_TOKEN_5_0 which are exported in common.yml